### DISCUSSION TITLE
Slingbox endpoint staying up?

### DISCUSSION THREAD

**Discussion by larsenv** [GerryDazoo/Slinger#228] [General]
We have a family friend named Dave, which is the reason why we have a Slingbox. He used to work for the company.

He said he's been observing GitHub repositories for Slingbox stuff. He said he was talking to their old founder "about what to tell our friends" and "asked [his] old dev team to look at Slinger". I'm not making this up.

With any luck, the server can still be up to obtain the password.

Dave, if you're reading this, sorry I didn't consult you before making a post regarding this.

**Comment by GerryDazoo** [comment]
Does Dave's last name happen to start with a "Z"? I talked with a Dave Z. yesterday. Maybe the same guy. This is good news if it does pan out.

**Reply by DoogieHowitzerWMD** [reply]
> Does Dave's last name happen to start with a "Z"? I talked with a Dave Z. yesterday. Maybe the same guy. This is good news if it does pan out.

Hey @GerryDazoo, assuming this did pan out, how would this impact setting up a remote? Would getting the remotes working just take some effort on our end to build a compatible remote file based on some default set of keycodes, or would this be a significant impediment?

**Comment by larsenv** [comment]
No, his name isn't Dave Zatz, assuming that's who you talked to. His name is Dave Mathews. I think he's met Dave Zatz.

**Reply by regmar505** [reply]
First, I am really grateful to Gerry about his effort to keep Slingbox alive. Can someone explains why the old Slingbox dev team does not release (Open Source) code for SlingPlayer apps for the Firestick or any other devices? It would be so cool to integrate SlingPlayer with Gerry's code.

**Reply by AndyJStubbs** [reply]
Interestingly (or mysteriously) my installed slingplayer windows app is currently still working as of this morning (10th November).

**Reply by floormanager1** [reply]
mine just stopped at 8am GMT

**Comment by floormanager1** [comment]
Link to the passwords has stopped working too.

**Reply by as400jockey** [reply]
These passwords - are you referring to my slingbox account password?  Or was this some unique password for each box?

I have my slingbox account password... but nothing specific for my two M1s, my Solo, and my original Slingbox.  Am I done for?

I still have SlingPlayer installed on a couple PCs (and they're still connecting OK) and my phone.  I can still connect from inside the house, on the same network.  Have not tried from outside.

If I need password for each box, can I retrieve it from one of the PCs?

**Reply by AndyJStubbs** [reply]
Yup mines still working at the moment too.

![image](https://user-images.githubusercontent.com/117518268/201151809-d0cba4b3-62bb-4036-9478-0096b955d90c.png)

**Comment by as400jockey** [comment]
Should have posted this as a new message, not a reply.... sorry... newbie here.

These passwords - are you referring to my slingbox account password? Or was this some unique password for each box?

I have my slingbox account password... but nothing specific for my two M1s, my Solo, and my original Slingbox. Am I done for?

I still have SlingPlayer Desktop installed on a couple PCs (and they're still connecting OK) and my phone. I can still connect from inside the house, on the same network. Have not tried from outside.

If I need password for each box, can I retrieve it from one of the PCs?

I should specify - I have Slingplayer Desktop running on several PCs, and the Slingplayer app running on a couple Android devices.  Still connecting internally...

**Comment by jkaponis** [comment]
try to get the password from https://newwatchsecure.slingbox.com/watch/slingAccounts/account_boxes_js but may be too late

**Reply by as400jockey** [reply]
the link is dead.  I found my admin password for my original slingbox (thing is a million years old and still works, although std def) an my Sling Solo.  Not sure if configuring later on with the newer desktop software would have overwritten the password.

But I don't have that info for my two M1s.

**Reply by jkaponis** [reply]
if you can still connect to m1 on pc , maybe run chrome devtools on the slingplayer and try to find you password

**Comment by tom76017** [comment]
I found my M1's admin password in **_C:\Users\\%username%\AppData\Roaming\Sling Media\Slingplayer Desktop\Logs\console.log_**

My M1 is running fine!

**Reply by GerryDazoo** [reply]
Andy, can you confirm your working box is not a Solo or Pro model. So far all the reports of bricked boxes have been for these older models.

**Reply by as400jockey** [reply]
Outstanding!  

I found the file.  I found an entry for each of my boxes.  For my original slingbox I see 
"username":"admin","password":"E1:51793..."

Is that encrypted?  Two of my boxes (original slingbox and sling solo) have the same admin password, and the E1: blurb is the same.  These are admin passwords I think I set up long ago with an early version of the Slingplayer desktop software.

For the other two, the M1s, the E1:yaddayadda...  are different from all the others.

Do I use this 'password' as is, with or without the E1, or do I need to decrypt somehow?

**Reply by AndyJStubbs** [reply]
My box is an M1 Gerry

**Reply by floormanager1** [reply]
My M1 and 350 still work (now disconnected). My Pro, Solo and ProHD are bricked.

**Reply by AndyJStubbs** [reply]
I wonder if the older box's don't store all of the config locally so depend upon the sling servers providing it to function . Guess in a few days we will know for sure!

**Reply by as400jockey** [reply]
Slingplayer on my android phone worked this morning, inside the house on the local network.  No longer.

Slingplayer desktop on my PC worked this morning.  Was still connected and playing the TV stream just now.  I closed and reopened.

THE ADS STILL WORK, but I'm not getting connected to the slingbox.

![image](https://user-images.githubusercontent.com/117923177/201173430-39b966d0-e9ce-46a1-8f81-40e62eab0ec0.png)

So that answers that question... I was hoping it would still work on the local network.  I have a Vulkano (predecessor to Slingbox) that still works on the local network.  As long as I never try to reconfigure the Vulkano I can still use it.

So no reason not to try this new Slinger gadget...  

New toy to play with.

**Reply by floormanager1** [reply]
> I wonder if the older box's don't store all of the config locally so depend upon the sling servers providing it to function . Guess in a few days we will know for sure!

I was hoping the direct connection option, which I had configured, would carry on working post Nov 9th.  Did not think they would destroy our property.

**Reply by as400jockey** [reply]
TOM76017 - what is the format of the password?  I find two things labeled "password" in that config file.  One looks like this:

..."password":"E1:11D0C084938FD4D9C6BFD5A377C1BFACC711BA0ED151CFC0AAE81EBC101C5236","name":"M1Basement"...

and the other looks like this:

..."name":"M1Basement","username":"admin","password":"9434q14011f629k"...

And what does your config.ini look like?  In some cases I see double quotes and in other cases I don't...

Tried both with no joy, but may have other issues as well...

**Reply by jkaponis** [reply]
is a 15 character password. must be the second one
password=9434q14011f629k

**Comment by as400jockey** [comment]
and now 20 minutes later it allowed me to connect again. Go figure.  And TV guide listings are not working, not that I care... just curious.

**Comment by Toei79** [comment]
in my case it looks kinda like hdhomerun, in someway , i add streamlink to  tivimate. its very cool have the stream link independent of any app.

any use on linux yea probably, question you can create one instruction for crontab -e so  its working every time on the background? i try wiht other stuff close to this one but its not working so far.

**Reply by GerryDazoo** [reply]
The slingbox.service is what you need to setup.

**Reply by Toei79** [reply]
what do this sorry im not expert , i able to run with one py  file, starting this one make run forever i mean even restart
